api/proto/teleport/plugins/v1/plugin_service.proto

```diff
@@ -43,9 +43,18 @@ message CreatePluginRequest {
   // In the scope of processing this request, these are exchanged for
   // short-lived renewable credentials, which are stored in the Plugin.
   types.PluginBootstrapCredentialsV1 bootstrap_credentials = 2;
-  // Static credentials are optional static credentials
-  // to supply to the plugin.
+
+  // StaticCredentials are an optional static credential to supply to the plugin.
   types.PluginStaticCredentialsV1 static_credentials = 3;
+
+  // StaticCredentials are an optional collection of static credentials
+  // to supply to the plugin. Ignored if `StaticCredentials` is set.
+  repeated types.PluginStaticCredentialsV1 static_credentials_list = 4;
+
+  // CredentialLabels are a collection of labels used to identify the
+  // credentials secified in the StaticCredentialsList. Ignored if
+  // StaticCredentials is used
+  map<string, string> credential_labels = 5;
```

Comment:
I'm not a fan of proto maps, so I'd much rather do `repeated Label credential_labels = 5;`, where Label is something like this:

```proto
message Label {
  string key = 1;
  string value = 2;
}
```

The API future proofing of maps is rather bad.

That said, we have plenty of labels defined like this, so I'll leave this up to you.

Reply:
I definitely see your argument re: `repeated Label`, but given that pretty much everything else encodes labels as a `map<string,string>` I'm going to go with consistency over "correctness" (if that's even the right word?) in this case.